test/unittests/commands/xmllib/test_values.py

```diff
@@ -0,0 +1,278 @@
+import warnings
+
+import pytest
+from lxml import etree
+
+from dsp_tools.commands.xmllib.models.values import BooleanValue
+from dsp_tools.commands.xmllib.models.values import ColorValue
+from dsp_tools.commands.xmllib.models.values import DateValue
+from dsp_tools.commands.xmllib.models.values import DecimalValue
+from dsp_tools.commands.xmllib.models.values import GeonameValue
+from dsp_tools.commands.xmllib.models.values import IntValue
+from dsp_tools.commands.xmllib.models.values import LinkValue
+from dsp_tools.commands.xmllib.models.values import ListValue
+from dsp_tools.commands.xmllib.models.values import Richtext
+from dsp_tools.commands.xmllib.models.values import SimpleText
+from dsp_tools.commands.xmllib.models.values import TimeValue
+from dsp_tools.commands.xmllib.models.values import UriValue
+from dsp_tools.models.custom_warnings import DspToolsUserWarning
+
+
+class TestBooleanValue:
+    def test_good(self) -> None:
+        with warnings.catch_warnings(record=True) as caught_warnings:
+            BooleanValue("False", ":booleanProp", resource_id="res_id")
+        assert len(caught_warnings) == 0
+
+    def test_warns(self) -> None:
+        with pytest.warns(DspToolsUserWarning):
+            BooleanValue("other", ":booleanProp", resource_id="res_id")
+
+    def test_serialise(self) -> None:
+        v = BooleanValue("0", ":booleanProp", resource_id="res_id")
+        expected = (
+            b'<boolean-prop xmlns="https://dasch.swiss/schema" '
+            b'xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name=":booleanProp">'
+            b'<boolean permissions="prop-default">false</boolean></boolean-prop>'
```

Comment:
If you want to stick with the XML string, a common thing you see in HTML formatting is to have each attribute on a new line with indentation. This could be emulated here:
```python
        b'<boolean-prop '
        b'    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" '
        b'    name=":booleanProp" '
        b'>'
```
I don't think it would semantically change anything, and to me it seems a lot more readable.

Reply:
I like Balduin's suggestion and have done something like that, because I have not been able to add indentation, because it considers that to be part of the string and the tests fail.

Reply:
Haven't you considered comparing the etree elements, instead of comparing strings? If I remember correctly, Balduin would be rather in favor of comparing etree elements, because it was him who once taught me this ;-)

Reply:
We can do either. But I would like to merge this soon. Since we are not yet finished anyway, we can always clean this up later.

Reply:
I clearly prefer comparing etree elements. These multi-line strings without semantic line breaks are ugly to look at, and you will always have troubles with XML-strings. But feel free to do it now or note it down for later, so that you are not blocked on your way forward!